charts/langsmith/templates/_helpers.tpl

```diff
@@ -200,6 +200,20 @@ Template containing common environment variables that are used by several servic
       key: openai_api_key
 - name: GO_ENDPOINT
   value: http://{{- include "langsmith.fullname" . }}-{{.Values.platformBackend.name}}:{{ .Values.platformBackend.service.port }}
+{{- if .Values.config.ttl }}
```

Comment:
since there are default values, can we remove the ifs?

Reply:
Yeah - maybe leave the if for the duration map?  Also, how about backwards compability?  If they don't have these it will fail without the if

Reply:
OK just left it for the duration map